File: kag_tools/inc/nav_item.cs
using kag_tools_shared;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace kag_tools.inc
{

    class nav_itemlist
    {
        public static List<nav_item> get_navs()
        {
            List<nav_item> nav_items = new List<nav_item>();
            //,U+E10F / Home
            //,U+E70F / Edit
            nav_items.Add(new nav_item { nav_name = "开始", nav_glyph = "", nav_page = typeof(Start) });
            nav_items.Add(new nav_item { nav_name = "文本编辑", nav_glyph = "", nav_page = typeof(Edit_ks) });
            return nav_items;
        }
    }
}

File: kag_tools_lagacy/inc/nav_item.vb
Imports kag_tools_shared

Public Class nav_itemlist


    Public Shared Function get_navs() As List(Of nav_item)
        Dim nav_items As New List(Of nav_item)()
        ',U+E10F / Home
        ',U+E70F / Edit
        nav_items.Add(New nav_item With {.nav_name = "开始", .nav_glyph = "", .nav_page = GetType(kag_tools_ui_down.Start)})
        nav_items.Add(New nav_item With {.nav_name = "文本编辑", .nav_glyph = "", .nav_page = GetType(kag_tools_ui_down.Edit_ks)})
        Return nav_items
    End Function

End Class
